src/pymovements/_utils/_html.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2025 The pymovements Project Authors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Provides functions for generating HTML representations of objects for Jupyter notebooks."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from html import escape
+from itertools import islice
+from typing import TypeVar
+from uuid import uuid4
+
+import polars as pl
+
+
+STYLE = """
+<style>
+    .pm-section-list {
+        margin: 0;
+        padding: 0;
+        font-family: sans-serif;
+    }
+    .pm-section {
```

Comment:
it could be possible that a name like `pm-section-list` could conflict with another css definition.

let's use the more verbose `pymovements-section` here and for all other occurrences.

Reply:
done